### PR TITLE
fix some typing mistakes

### PR DESCRIPTION
**Pull request by akkspros**
fix some typing mistakes

### CODE REVIEW

**Comment by akkspros** [comment]
@yoavf can you check this please

**Review by pedro-mendonca — APPROVED** [review]

**Comment by toolstack** [comment]
That's a strange failure on TravisCI, I've restarted the builds to see if it was just temporary, if not I'll take a look in to it some more.

File: assets/js/vendor/jquery.tablesorter.js [hunk 1]
```diff
@@ -697,11 +697,11 @@
 							extractor = ts.getParserById( ts.getData( header, configHeaders, 'extractor' ) );
 							parser = ts.getParserById( ts.getData( header, configHeaders, 'sorter' ) );
 							noParser = ts.getData( header, configHeaders, 'parser' ) === 'false';
-							// empty cells behaviour - keeping emptyToBottom for backwards compatibility
+							// empty cells behavior - keeping emptyToBottom for backwards compatibility
```
Comment (toolstack) [inline assets/js/vendor/jquery.tablesorter.js:700]: Not going to block it because of these, but behaviour is a correct spelling... in Canada ;)

**Review by toolstack — APPROVED** [review]
+1

File: assets/js/vendor/jquery.tablesorter.js (hunk 1 above)
Comment (ocean90) [reply assets/js/vendor/jquery.tablesorter.js:700]: Third-party libraries shouldn't be changed at all and reported upstream instead.

Comment (toolstack) [reply assets/js/vendor/jquery.tablesorter.js:700]: Agreed, it didn't even click that it was in a library.  There is a new version of the library so I'll create an issue/PR to update it.

**Pull request merged**